app/backend/graphql/task/mutations/createTaskMutationField.ts

```diff
@@ -0,0 +1,29 @@
+import { arg, intArg, mutationField, nullable, stringArg } from 'nexus'
+import { DateScalar } from '../../scalars/date'
+import { Task } from '../task'
+
+export const createTaskMutationField = mutationField('createTaskMutation', {
```

Comment:
Let's name the mutation `taskCreate`.

By using the prefix "task" all task related mutations are grouped together. Also the suffix "mutation" does not help.

Reply:
done